flume-ng-sinks/flume-ng-elasticsearch-sink/src/main/java/org/apache/flume/sink/elasticsearch/client/ElasticSearchClientFactory.java

```diff
@@ -20,14 +20,16 @@
 
 import org.apache.flume.sink.elasticsearch.ElasticSearchEventSerializer;
 import org.apache.flume.sink.elasticsearch.ElasticSearchIndexRequestBuilderFactory;
+import static org.apache.flume.sink.elasticsearch.ElasticSearchSinkConstants.TRANSPORT_CLIENT;
+import static org.apache.flume.sink.elasticsearch.ElasticSearchSinkConstants.REST_CLIENT;
 
 /**
  * Internal ElasticSearch client factory. Responsible for creating instance
  * of ElasticSearch clients.
  */
 public class ElasticSearchClientFactory {
-  public static final String TransportClient = "transport";
-  public static final String RestClient = "rest";
+  public static final String TransportClient = TRANSPORT_CLIENT;
+  public static final String RestClient = REST_CLIENT;
```

Comment:
Note to reviewers: these seem to be used in quite a few places.  Preserving the original names makes the diff smaller.  Let me know if you prefer that I replace the names instead.